lib/ansible/modules/web_infrastructure/ansible_tower/tower_settings.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2018, Nikhil Jain <nikjain@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: tower_settings
+author: "Nikhil Jain (@jainnikhil30)"
+version_added: "2.7"
+short_description: Get, Modify Ansible Tower settings.
+description:
+    - Get, Modify Ansible Tower settings. See
+      U(https://www.ansible.com/tower) for an overview.
+options:
+    name:
+      description:
+        - Name of setting to get/modify.
+    value:
+      description:
```

Comment:
My concern about this really needing to be two modules, one for settings_facts and one for settings, has not been addressed.

Reply:
@ryansb I have made this change. This module now only allows to modify the settings. I have removed the "get" part. I will follow it up with a _facts module to adhere with core best practices. Apologies to have not done it right way before.